types/ffmpeg/index.d.ts

```diff
@@ -0,0 +1,162 @@
+    genre: string;
+    copyright: string;
+    description: string;
+    synopsis: string;
+    show: string;
+    episode_id: string;
+    network: string;
+    lyrics: string;
+}>;
+
+type WMVMetadata = Partial<{
+    title: string;
+    author: string;
+    copyright: string;
+    comment: string;
+    rating: string;
+}>;
+
+type AVIMetadata = Partial<{
+    IARL: string;
+    IART: string;
+    ICMS: string;
+    ICMT: string;
+    ICOP: string;
+    ICRD: string | Date;
+    ICRP: string;
+    IDIM: string;
+    IDPI: string;
+    IENG: string;
+    IGNR: string;
+    IKEY: string;
+    ILGT: string;
+    ILNG: string;
+    IMED: string;
+    INAM: string;
+    IPLT: string;
+    IPRD: string;
+    ISBJ: string;
+    ISFT: string;
+    ISHP: string;
+    ISRC: string;
+    ISRF: string;
+    ITCH: string;
+}>;
+
+type FLVMetadata = Partial<{
+    duration: number;
+    filesize: string;
+    encoder: string;
+    width: number;
+    height: number;
+    videodatarate: number;
+    videocodecid: string;
+    audiodatarate: number;
+    audiosamplerate: number;
+    stereo: boolean;
+    audiocodecid: string;
+}>;
+
+interface Video {
+    addCommand(command: string, argument: string): void;
+    addInput(argument: string): void;
+    addFilterComplex(argument: string): void;
+    setOutput(path: string): void;
+    setDisableAudio(): Video;
+    setVideoFormat(format: string): Video;
+
+    setVideoCodec(codec: string): Video;
+    setVideoBitRate(bitrate: number): Video;
+    setVideoFrameRate(framerate: number): Video;
+
+    /**
+     *
+     * @description time in seconds
+     * @example 13
+     */
+    setVideoStartTime(time: number | string): Video;
+
+    /**
+     *
+     * @example 13
+     */
+    setVideoDuration(duration: number | string): Video;
+
+    /**
+     *
+     * @example 1.77
+     */
+    setVideoAspectRatio(aspect: number | string): Video;
+
+    setVideoSize(size: string, keepPixelAspectRatio: boolean, keepAspectRatio: boolean, paddingColor?: string): Video;
+
+    setAudioCodec(codec: string): Video;
+
+    setAudioFrequency(frequency: number): Video;
+    setAudioChannels(channel: number): Video;
+
+    setAudioBitrate(bitrate: number): Video;
+
+    setAudioQuality(quality: number): Video;
+
+    setWatermark(watermarkPath: string, settings: WatermarkSettings): Video;
+
+    save(destinationFileName: string, callback: SaveCallback): void;
+
+    fnExtractSoundToMP3(destinationFileName: string, callback: SaveCallback): void;
+
+    fnExtractFrameToJPG(destinationFolder: string, settings: FrameToJPGSettings, callback: SaveCallback): void;
+
+    fnAddWatermark(watermarkPath: string, newPilePath: string, settings: WatermarkSettings, callback: SaveCallback): void;
+
+    metadata: StandardVideoMetadata & AVIMetadata & FLVMetadata;
+}
+
+interface Iffmpeg {
+    new(filePath: string, cb?: (err: Error, video: Video) => void): Promise<Video>;
+}
+
+declare var ffmpeg: Iffmpeg;
+
+export default ffmpeg;
```

Comment:
This needs `export =` instead of `export default`. Refer to https://github.com/DefinitelyTyped/DefinitelyTyped#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default